AnnoyancesFilter/sections/popups.txt

```diff
@@ -104,6 +104,8 @@ academictimes.com###cover
 stuff.co.nz##body > div.t5.t6
 economist.com###piano-ribbon
 fromsmash.com###welcome-modal
+teknikhukuk.com#$#.g1-popup { display: none; !important }
+teknikhukuk.com#$#body { overflow: visible; !important; }
```

Comment:
Unnecessary semicolons before the exclamation points, and please change body to html